src/screens/poa-form/index.js

```diff
@@ -60,21 +60,21 @@ class PoAForm extends Component<PoAFormProps, PoAFormState> {
         name: '',
         street_address: '',
         locality: '',
-        region: 'TN',
+        region: '',
```

Comment:
We want to leave these for now. There was a UX decision to pre-populate these fields with TN because the form is currently only for TN residents.

Reply:
For now we can leave the blank option because the user will need to intentionally select it in order to proceed and get a blank form. @lgonzo1105 may have an opinion on whether we go ahead and let a user submit a form with everything blank but `TN` as the selected state or exactly how this user interaction should work.